Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -2692,3 +2692,473 @@ kraken,ZRXUSD,crypto,ZRX/USD,USD,1,0.001,0.00000001,ZRXUSD,5
 kraken,ZRXBTC,crypto,ZRX/XBT,BTC,1,0.00000001,0.00000001,ZRXXBT,5
 kraken,USDCAD,crypto,USD/CAD,CAD,1,0.00001,0.00000001,ZUSDZCAD,10
 kraken,USDJPY,crypto,USD/JPY,JPY,1,0.001,0.00000001,ZUSDZJPY,10
+
+ftx,1INCHUSD,crypto,1INCH/USD,USD,1,0.0001,0.00000001,1INCH/USD
```

Comment:
Does `FTX` have a minimum order size per symbol like the other crypto brokerages?